### PR TITLE
[issue-5952] upgrade to jetty 9.4.44.v20210927

### PR DESCRIPTION
**Pull request by pjfanning**
### Description

This PR...

Fixes: #5952

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [X] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -145,8 +145,8 @@
         <cs.jaxb.version>2.3.0</cs.jaxb.version>
         <cs.jaxws.version>2.3.2-1</cs.jaxws.version>
         <cs.jersey-client.version>2.26</cs.jersey-client.version>
-        <cs.jetty.version>9.4.36.v20210114</cs.jetty.version>
-        <cs.jetty-maven-plugin.version>9.4.27.v20200227</cs.jetty-maven-plugin.version>
+        <cs.jetty.version>9.4.44.v20210927</cs.jetty.version>
+        <cs.jetty-maven-plugin.version>9.4.44.v20210927</cs.jetty-maven-plugin.version>
```
Comment (rohityadavcloud) [inline pom.xml:149]: I think the mvn plugin is for developers only, there are some issues because of which we still use 9.4.27... cc @sureshanaparti

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by rohityadavcloud** [comment]
@pjfanning see Travis for build failures

**Comment by rohityadavcloud** [comment]
The jetty mvn plugin is used only by developer to start/stop mgmt server. The jetty depedency howerver is used by mgmt server.
@blueorangutan package

**Comment by rohityadavcloud** [comment]
@blueorangutan test

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
cc @sureshanaparti to check/consider if it make sense to upgrade jetty (I checked it was just minor release of jetty 9.4) in minor release

**Review by sureshanaparti — APPROVED** [review]

**Pull request merged**